shaders/src/light_punctual.fs

```diff
@@ -30,9 +30,15 @@ uvec3 getFroxelCoords(const highp vec3 fragCoords) {
     froxelCoord.xy = uvec2(fragCoords.xy * frameUniforms.resolution.xy *
             vec2(frameUniforms.oneOverFroxelDimension, frameUniforms.oneOverFroxelDimensionY));
 
-    froxelCoord.z = uint(max(0.0,
-            log2(frameUniforms.zParams.x * fragCoords.z + frameUniforms.zParams.y) *
-                    frameUniforms.zParams.z + frameUniforms.zParams.w));
+    // go from screen-space (non-inverted) Z to normalized view-space Z (i.e. scaled by 1/zLightFar)
+    // (see Froxelizer.cpp)
+    highp float viewSpaceNormalizedZ = frameUniforms.zParams.x * fragCoords.z + frameUniforms.zParams.y;
+    // compute the sliceZ mapping
+    highp float sliceZ = log2(viewSpaceNormalizedZ) * frameUniforms.zParams.z + frameUniforms.zParams.w;
```

Comment:
Does the slice need to be highp? That seems unnecessary

Reply:
yes good point